Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -2,54 +2,54 @@
 --exclude Carthage,Pods,.build
 
 # options
---swiftversion 6.0
---languagemode 5
+--swift-version 6.0
+--language-mode 5
 --self remove # redundantSelf
---importgrouping testable-bottom # sortedImports
---commas always # trailingCommas
---trimwhitespace always # trailingSpace
+--import-grouping testable-bottom # sortedImports
+--trailing-commas always # trailingCommas
```

Comment:
@calda is it intentional that this option name changed?

Reply:
yeah, the `--commas` rule was renamed to `--trailing-commas` https://github.com/nicklockwood/SwiftFormat/blob/38c10de70c92b1a34c4e3afb6c082be3e53d4a1a/Sources/OptionDescriptor.swift#L1368-L1379